src/actions/authActions.js

```diff
@@ -493,9 +492,6 @@ export const resetAppServicesAction = () => {
     // reset data stored in keychain
     await resetKeychainDataObject();
 
-    // clear images cache
-    ImageCacheManager().clearCache().catch(() => null);
```

Comment:
FastImage does not contain method for forcing clearing cache. This should not be a problem, as AFAIK our images do not contain private information.